app/assets/scss/partials/_app.scss

```diff
@@ -46,6 +53,7 @@
     content: " ";
     display: table;
     width: 100%;
+    height: 100%;
```

Comment:
@hannalaakso Ah I thought that was a bug!

The blue outline now goes all the way around the iframe.

Because `iFrameResizer` can't calculate the height of a `position: fixed` modal, the fixed height comes from this bit that I've hard-coded:

```css
// Allow fixed height for "position: fixed" modal
.app-iframe-in-component-preview--modal {
  height: 400px;

  @include govuk-media-query ($from: tablet) {
    height: 500px;
  }
}
```

Reply:
Going to resolve this as it doesn't concern the component itself. Good to have the conversation here for reference though.